### PR TITLE
Update News.js

### PR DESCRIPTION
**Pull request by Kengetsu**
Fix for when no posts exist in the database

### CODE REVIEW

**Comment by HitoriWoB** [comment]
Did you edit the build file directly for this change? If you did, we will want to change the method in which this is achieved. You should be editing the ui_component src files and your file watcher should update the build files. 

If you're not sure how this is done, the section on Advanced UI in the main README file has more details.

**Comment by Kengetsu** [comment]
Ah sorry, I edited the file directly in github since my ide was behind a bunch lol. Must of accidentally picked the build version. Let me fix it. lol

**Comment by aumu1031** [comment]
LGTM, adding Lsm as reviewer

File: ui_components/src/home/News.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ type Props = {|
     },
 |};
 export function News({ initialNewsPosts, isAdmin, version, homeLinks }: Props): React$Node {
-    const [activePostId, setActivePostId] = React.useState(initialNewsPosts[0] !== "undefined" ? initialNewsPosts[0].post_id : null);
+    const [activePostId, setActivePostId] = React.useState(initialNewsPosts[0] !== undefined ? initialNewsPosts[0].post_id : null);
```
Comment (levimeahan) [inline ui_components/src/home/News.js:16]:
```suggestion
    const [activePostId, setActivePostId] = React.useState(initialNewsPosts[0]?.post_id || null);
```

Comment (levimeahan) [reply ui_components/src/home/News.js:16]: This should work if we want to be more concise

**Review by levimeahan — APPROVED** [review]

**Pull request merged**